index.bs

```diff
@@ -874,17 +871,36 @@ Given a [=request=] |request|:
 
 <h3 dfn id="does-filter-data-match">Does filter data match</h3>
 
+To <dfn>match [=filter values=]</dfn> given a [=filter value=] |a| and a [=filter value=] |b|:
+1. If |b| [=list/is empty=], then:
```

Comment:
For clarity, should we replace all these `[=list/is empty=]` with `[=set/is empty=]`? Not sure if the links work correctly, though.

Reply:
Yes, [=set/is empty=] is more clear and the link works. Just updated. Thanks.